sklearn/ensemble/_forest.py

```diff
@@ -2653,3 +2654,21 @@ def transform(self, X):
         """
         check_is_fitted(self)
         return self.one_hot_encoder_.transform(self.apply(X))
+
+    def get_feature_names_out(self, input_features=None):
+        """Get output feature names for transformation.
+        Parameters
+        ----------
+        input_features : array-like of str or None, default=None
+            Not used, present here for API consistency by convention.
+        Returns
+        -------
+        feature_names_out : ndarray of str objects
+            Transformed feature names.
+        """
```

Comment:
This should hopefully fix the numpydoc validation test:

https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=34120&view=logs&j=78a0bf4f-79e5-5387-94ec-13e67d216d6e&t=dbc16e8a-e84c-5cb6-14f6-590d872d6e0b&l=98


```suggestion
        """Get output feature names for transformation.

        Parameters
        ----------
        input_features : array-like of str or None, default=None
            Not used, present here for API consistency by convention.
  
        Return
        ------
        feature_names_out : ndarray of str objects
            Transformed feature names.
        """
```